packages/storybook/src/executors/models.ts

```diff
@@ -6,14 +6,34 @@ export interface StorybookConfig {
 }
 
 export interface CommonNxStorybookConfig {
-  uiFramework:
-    | '@storybook/angular'
-    | '@storybook/react'
-    | '@storybook/html'
-    | '@storybook/web-components'
-    | '@storybook/vue'
-    | '@storybook/vue3'
-    | '@storybook/svelte'
-    | '@storybook/react-native';
+  uiFramework: UiFramework;
+  uiFramework7: UiFramework7;
   config: StorybookConfig;
 }
+
+export type UiFramework7 =
```

Comment:
[@storybook/next](https://github.com/storybookjs/storybook/tree/next/code/frameworks/nextjs) is missing. It is a relatively new Storybook framework to support Storybook for Next.js projects without requiring too much custom config from the user's side. We would recommend using `@storybook/next` over `@storybook/react-<webpack5|vite>` in Next.js projects. I can assist in supporting it in `@nrwl/storybook` if you need any help.

Reply:
Thanks @valentinpalkovic !

Reply:
I got the list from here: https://github.com/storybookjs/storybook/blob/next/MIGRATION.md#framework-field-mandatory

So, maybe the `@storybook/next` is missing from that list?

Reply:
Seems so. Let's add it there. I will take care of it! Thanks for letting us know :)